app/assets/stylesheets/content/_select2.scss

```diff
@@ -200,3 +195,7 @@ $se2-width: 100%;
     }
   }
 }
+
+input[type="text"].select2-input {
```

Comment:
This bug is reproduced only in chrome as far as I know, maybe in future versions this thing will be resolved or in foundation source.